src/app/routes/loadInitialData/index.js

```diff
@@ -2,16 +2,10 @@
  * © Jordan Tart https://github.com/jtart
  * https://github.com/jtart/react-universal-app
  */
-import { matchRoutes } from 'react-router-config';
+import getRouteProps from '../getInitialData/utils/getRouteProps';
 
 const loadInitialData = async (url, routes) => {
```

Comment:
I have proposed to completely remove this method and move the remaining login into the main App class as a part of another refactor: https://github.com/bbc/simorgh/issues/1676